metaversefile-api.js

```diff
@@ -387,6 +387,9 @@ metaversefile.setApi({
   usePostPerspectiveScene() {
     return postScenePerspective;
   }, */
+  getReverbZone() {

```

Comment:
`getReverbZones()`

Reply:
[change the reverbZone to reverbZones](https://github.com/webaverse/app/pull/3056/commits/84a5bd2a924343dc99e8f7828d5f14cf85cd4958)